src/data/detail-page/charts/devOps.ts

```diff
@@ -1,4 +1,61 @@
-import { Chart } from './chartInterfaces';
+import { Chart, ChartDataItem, ChartColor } from './chartInterfaces';
```

Comment:
I wonder if we should not move the `charts` data folder one level up (from `detail-page` to `data`). Charts are now part of our DetailPage, but they are a nice, separate entity that could be used (or moved) anywhere. What do you think?

Reply:
I moved it one level up. Good idea.